students/jongho/users/views.py

```diff
@@ -1,3 +1,38 @@
-from django.shortcuts import render
+import json,re
 
-# Create your views here.
+from django.views import View
+from django.http  import JsonResponse
+from django.db    import DataError
+
+from .models      import User
+
+class SignUp(View):
+  def post(self,request):
+    try:
+      data                = json.loads(request.body)
+      EMAIL_REGEX         = '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
+      PSSWORD_REGEX       = '^.*(?=^.{8,15}$)(?=.*\d)(?=.*[a-zA-Z])(?=.*[!@#$%^&+=]).*$'
+
+      if not re.match(EMAIL_REGEX,data["email"]):
+        return JsonResponse({'message':"INVALID_EMAIL"},status=400)
+      
+      if not re.match(PSSWORD_REGEX,data["password"]):
+        return JsonResponse({'message':"INVALID_PASSWORD"},status=400)
+
+      if User.objects.filter(email=data["email"]).exists():
+        return JsonResponse({'message':"INVALID_EGTIKEUL"},status=400)
```

Comment:
작성해주신 if문은 이미 존재하는 이메일인지 확인하는 로직입니다. 그에 맞는 에러 메세지로 작성해주세요.
```suggestion
      if User.objects.filter(email=data["email"]).exists():
        return JsonResponse({'message':"email_already_exists"},status=400)
```